### PR TITLE
Update/homunculus_property

### PR DESCRIPTION
**Pull request by sctnightcore**
```
 homunculus_property
 /// Sends information about owned homunculus to the client . [orn]
/// 022e <name>.24B <modified>.B <level>.W <hunger>.W <intimacy>.W <equip id>.W <atk>.W <matk>.W <hit>.W <crit>.W <def>.W <mdef>.W <flee>.W <aspd>.W <hp>.W <max hp>.W <sp>.W <max sp>.W <exp>.L <max exp>.L <skill points>.W <atk range>.W	(ZC_PROPERTY_HOMUN)
/// 09f7 <name>.24B <modified>.B <level>.W <hunger>.W <intimacy>.W <equip id>.W <atk>.W <matk>.W <hit>.W <crit>.W <def>.W <mdef>.W <flee>.W <aspd>.W <hp>.L <max hp>.L <sp>.W <max sp>.W <exp>.L <max exp>.L <skill points>.W <atk range>.W (ZC_PROPERTY_HOMUN_2)

```

### CODE REVIEW

File: src/Network/Receive/ServerType0.pm [hunk 1]
```diff
@@ -582,6 +582,7 @@ sub new {
 		'09F2' => ['rodex_get_zeny', 'V2 C2', [qw(mailID1 mailID2 type fail)]],   # 12
 		'09F4' => ['rodex_get_item', 'V2 C2', [qw(mailID1 mailID2 type fail)]],   # 12
 		'09F6' => ['rodex_delete', 'C V2', [qw(type mailID1 mailID2)]],   # 11
+		'09F7' => ['homunculus_property', 'Z24 C v20 V2 v2', [qw(name state level hunger intimacy accessory atk matk hit critical def mdef flee aspd hp hp_max sp sp_max exp exp_max points_skill attack_range)]],
```
Comment (sctnightcore) [inline src/Network/Receive/ServerType0.pm]: v20 is work ? 

@alisonrag

Comment (alisonrag) [reply src/Network/Receive/ServerType0.pm]: yep

Comment (sctnightcore) [reply src/Network/Receive/ServerType0.pm]: v24 ? 

I'm not sure . 
I might count it wrong.

Comment (alisonrag) [reply src/Network/Receive/ServerType0.pm]: the count is wrong =p

Comment (alisonrag) [reply src/Network/Receive/ServerType0.pm]: Z24 C v12 V2 v2 V2 v2

Comment (sctnightcore) [reply src/Network/Receive/ServerType0.pm]: ```<level>.W <hunger>.W <intimacy>.W <equip id>.W <atk>.W <matk>.W <hit>.W <crit>.W <def>.W <mdef>.W <flee>.W <aspd>.W <hp>.W <max hp>.W <sp>.W <max sp>.W```

= v16
```
<level>.W <hunger>.W <intimacy>.W <equip id>.W <atk>.W <matk>.W <hit>.W <crit>.W <def>.W <mdef>.W <flee>.W <aspd>.W <hp>.L <max hp>.L <sp>.W <max sp>.W```

= v24 

?

Comment (sctnightcore) [reply src/Network/Receive/ServerType0.pm]: ```'022E' => ['homunculus_property', 'Z24 C v16 V2 v2', [qw(name state level hunger intimacy accessory atk matk hit critical def mdef flee aspd hp hp_max sp sp_max exp exp_max points_skill attack_range)]],```

Comment (alisonrag) [reply src/Network/Receive/ServerType0.pm]: L = A signed long (32-bit) value.
v =  An unsigned short (16-bit).

Comment (sctnightcore) [reply src/Network/Receive/ServerType0.pm]: Z24 C v24 V2 v2 !

**Comment by sctnightcore** [comment]
test with `Z24 C v12 V2 v2 V2 v2` it work . 


```
homun status
-------------------------------- Slave Status --------------------------------
Name: Filir                       HP: 3134/3134 (100.00%)
Type: Homunculus                  SP:   282/282 (100.00%)
Job:  High Filir
Level: 1                          Exp: 0/347 (0.00%)

Atk:    48     Matk:      104     Hunger:         32
Hit:   192     Critical:   13     Intimacy:       10
Def:   135     Mdef:       99     Accessory:       0
Flee:   52     Aspd:      147     Summons:       N/A
Range:   2     Skill pt:    0     Contract End:  N/A

Statuses:
------------------------------------------------------------------------------
```

**Pull request merged**